config/environments/development.rb

```diff
@@ -41,6 +41,18 @@
 
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+
+  config.action_mailer.delivery_method = :letter_opener
+
+  config.action_mailer.perform_deliveries = true
+
+  config.action_controller.raise_on_missing_callback_actions = true
+
+  LetterOpener.configure do |config|
+    config.browser = :chrome
+  end
```

Comment:
- [ ] It wasn't possible to open your app due to this issue, please 🙏🏽 correct this error to be able to check your app 👇🏽 

```ruby
bin/rails aborted!
NoMethodError: undefined method `browser=' for #<LetterOpener::Configuration:0x00007f54bcc002e8 @location=#<Pathname:/home/franclobo/microverse/eugene-mindscape/tmp/letter_opener>, @message_template="default", @file_uri_scheme=nil> (NoMethodError)

    config.browser = :chrome
          ^^^^^^^^^^
/home/franclobo/microverse/eugene-mindscape/config/environments/development.rb:53:in `block (2 levels) in <main>'
/home/franclobo/microverse/eugene-mindscape/config/environments/development.rb:52:in `block in <main>'
/home/franclobo/microverse/eugene-mindscape/config/environments/development.rb:3:in `<main>'
/home/franclobo/microverse/eugene-mindscape/config/environment.rb:5:in `<main>'
Tasks: TOP => db:drop => db:load_config => environment
(See full trace by running task with --trace)
```